js/jquery.emojiarea.js

```diff
@@ -702,9 +702,11 @@
 
 		// Note on the window global if the emoji picker is visible. Since there
 		// can be only one emoji picker at any time, this will work.
-		window.emojiPickerStatus.isPickerVisible = false;
+    updateGlobalEmojiPickerVisiblity(this, false);
 
 		this.$menu.hide("fast", function(){
+      updateGlobalEmojiPickerVisiblity(this, false);
```

Comment:
Why is this call to updateGlobalEmojiPickerVisiblity need to be made if it is made above on line 705?